companion/src/simulation/telemetrysimu.cpp

```diff
@@ -153,53 +161,78 @@ void TelemetrySimulator::onSimulateToggled(bool isChecked)
     stopTelemetry();
 }
 
-void TelemetrySimulator::onLogTimerEvent()
+TelemetryProvider * TelemetrySimulator::newTelemetryProviderFromDropdownChoice(const QString &text, QScrollArea * parent, bool isExternal)
 {
-  logPlayback->stepForward(false);
+  if (text == "None") {
```

Comment:
This an subsequent branch tests will not achieve the desired result when the text is translated in the drop down list. Use the currentIndex instead